### ISSUE TITLE
train, val, test splits of the dataset

### ISSUE THREAD

**Issue by auniquesun** [omniobject3d/OmniObject3D#36]
Thansks for sharing the paper and dataset. Excellent work!

May I ask are there official train, val and test splits of the OmniObject3D dataset. I want to use it to train and evaluate 3D point cloud recognition models, as the functionality of ModelNet40 and ScanObjectNN.

Thanks. Looking forward to your reply.

**Comment by wutong16** [comment]
Thank you for your interest in our work. For the point cloud perception benchmark in the paper, we utilized a network pre-trained on ModelNet40 to evaluate the OmniObject3D dataset, focusing exclusively on the shared categories. As a result, we did not differentiate between train / val / test splits; all data functioned solely as the test set in this specific context.

However, we highly encourage you to establish your own train / val / test splits if you plan to use this dataset for training point cloud recognition models. We are happy to integrate your configuration into this repository for follow-up works in this track.

**Comment by auniquesun** [comment]
> Thank you for your interest in our work. For the point cloud perception benchmark in the paper, we utilized a network pre-trained on ModelNet40 to evaluate the OmniObject3D dataset, focusing exclusively on the shared categories. As a result, we did not differentiate between train / val / test splits; all data functioned solely as the test set in this specific context.
> 
> However, we highly encourage you to establish your own train / val / test splits if you plan to use this dataset for training point cloud recognition models. We are happy to integrate your configuration into this repository for follow-up works in this track.

I got it. I will create train/val/test split by myself and let you know later. Thanks for your reply.

**Comment by auniquesun** [comment]
After confirmation, some categories contain only 2 samples, thus the the omniobject dataset can not be split into train/val/test, since we need at least 3 samples for each category.
